python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -124,6 +141,45 @@ def __init__(
             log: The logger instance to use. (Optional)
         """
 
+    @overload
+    def __init__(
+        self,
+        deployment_name: str,
+        endpoint: Union[HttpsUrl, str],
+        api_version: str = DEFAULT_AZURE_API_VERSION,
+        api_key: Optional[str] = None,
+        ad_token: Optional[str] = None,
+        ad_token_provider: Optional[AsyncAzureADTokenProvider] = None,
+        default_headers: Optional[Mapping[str, str]] = None,
+        log: Optional[Logger] = None,
+        use_extensions: bool = False,
+    ) -> None:
+        """
+        Initialize an AzureChatCompletion service.
+
+        Arguments:
+        deployment_name: The name of the Azure deployment. This value
+            will correspond to the custom name you chose for your deployment
+            when you deployed a model. This value can be found under
+            Resource Management > Deployments in the Azure portal or, alternatively,
+            under Management > Deployments in Azure OpenAI Studio.
+        endpoint: The endpoint of the Azure deployment. This value
+            can be found in the Keys & Endpoint section when examining
+            your resource from the Azure portal, the endpoint should end in openai.azure.com.
+        api_key: The API key for the Azure deployment. This value can be
+            found in the Keys & Endpoint section when examining your resource in
+            the Azure portal. You can use either KEY1 or KEY2.
+        api_version: The API version to use. (Optional)
+            The default value is "2023-05-15".
+        ad_auth: Whether to use Azure Active Directory authentication. (Optional)
+            The default value is False.
+        default_headers: The default headers mapping of string keys to
+            string values for HTTP requests. (Optional)
+        log: The logger instance to use. (Optional)
+        use_extensions: Whether to use extensions, for example when chatting with data. (Optional)
+            The default value is False.
```

Comment:
please add a note on what happens when this is set

Reply:
Updated docstring